platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -2089,8 +2089,7 @@
     <bundle>mvn:org.apache.camel/camel-schematron/${upstream.version}</bundle>
   </feature>
    <feature name='camel-script-jruby' version='${upstream.version}' resolver='(obr)' start-level='50'>
-    <feature version='${upstream.version}'>camel-script</feature>
-    <bundle>mvn:org.jruby/jruby-complete/${jruby.tesb.version}</bundle>
+    <!-- removed for security reasons -->
```

Comment:
I suggest to add to the comment: "CVE-2022-25857 (high, snakeyaml included in jruby) and further medium CVEs". This clarifies better while this feature has been cleared.

Reply:
Ok, done

Reply:
CVE reference is added.